### DISCUSSION TITLE
No full keyboard?

### DISCUSSION THREAD

**Discussion by electr1fy0** [AppGround-io/bluetooth-keyboard-and-mouse-support#282] [Q&A]
Am I missing something or there's no out of the box way to have the full keyboard be displayed with the modifier keys?
If not I'd love this feature to be added and I hope I'll be able to rename the keys for macOS.

**Comment by AdamKane41** [comment]
It uses your default Keyboard app instaled on your phone. To do what you want, you need to install a keyboard app that has all the keys a "ordinary phisycal" keyboard has, and set it as your active keyboard via your Phone's Android setting.

**Comment by AppGround-io** [comment]
A full PC keyboard control is available in the new version 4.25.0 of the app rolling out in the next days.
